CMakeLists.txt

```diff
@@ -1449,6 +1453,21 @@ if(WITH_GTK OR HAVE_GTK)
   endif()
 endif()
 
+if(WITH_FRAMEBUFFER OR HAVE_FRAMEBUFFER OR WITH_FRAMEBUFFER_XVFB OR HAVE_FRAMEBUFFER_XVFB)
+  if(HAVE_FRAMEBUFFER OR HAVE_FRAMEBUFFER_XVFB)
+    if(HAVE_FRAMEBUFFER)
+      status("    Framebuffer UI:" "YES")
+    endif()
+    if(HAVE_FRAMEBUFFER_XVFB)
+      status("    Virtual framebuffer UI:" "YES")
+    else()
+      status("    Virtual framebuffer UI:" "NO")
```

Comment:
No need to extra `if` blocks.
Just take a look on the lines below this inserted code.

Reply:
fixed